frontend/package.json

```diff
@@ -22,7 +23,20 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "eslint 'src/**/*.ts*' && prettier --check 'src/**/*.ts*'",
```

Comment:
On my setup (maybe a windows-only problem?) the quotes around the pattern led to the following error: 
"No files matching the pattern "'src/**/*.ts*'" were found.
Please check for typing mistakes in the pattern."
Seems like the quotes are recognized as part of the pattern. 
This could be resolved by removing the quotes around the pattern.

Reply:
I'm on Linux and there's no error for me. I can remove the quotes, though.

Reply:
@jannikwinghart removed now, can you check if it works on your end?